fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1215,6 +1218,20 @@ public void waitForReady() throws InterruptedException {
 
             Thread.sleep(2000);
             LOG.info("wait globalStateMgr to be ready. FE type: {}. is ready: {}", feType, isReady.get());
+
+            if (System.currentTimeMillis() - waitStartTime > printHintTimeInterval * printHintCnt) {
+                printHintCnt++;
+                LOG.warn("It took too much time to get ready, there are some reasons: " +
+                        "1. There are too many BDB logs to replay, because the failure of checkpoint. " +
+                        "2. More than half of the followers in the cluster are not started. " +
+                        "3. There are multiple IPs on the machine, or ip has changed, " +
+                        "you should specify the priority_networks to use the IP from image/ROLE, " +
+                        "ignore this reason if you are using FQDN. " +
+                        "4. The time deviation between machines is greater than 5s, " +
+                        "please use ntp or other tools to synchronize clock. " +
+                        "5. The configuration of edit_log_port is changed, please reset to the original value. " +
+                        "6. The thread of replayer is blocked, please use jstack to find the details");
+            }
```

Comment:
7. The remaining capacity of disk where FE meta dir is on is under 5GB.

Reply:
If free disk size is less than 5G, FE will not start.